terraform/environments/mlra/modules/codebuild/main.tf

```diff
@@ -50,11 +63,12 @@ resource "aws_s3_bucket_versioning" "report_versioning" {
 ######################################################
 
 resource "aws_ecr_repository" "local-ecr" {
+  #checkov:skip=CKV_AWS_136:TODO Will be addressed as part of https://dsdmoj.atlassian.net/browse/LASB-3390
   name                 = "${var.app_name}-local-ecr"
-  image_tag_mutability = "MUTABLE"
+  image_tag_mutability = "IMMUTABLE"
 
   image_scanning_configuration {
-    scan_on_push = false
+    scan_on_push = true
```

Comment:
This is changing the way we push images to the image repo - although helpful to scan them - what will happen if they fail? Will it block a release? I'd keep this to a separate task outside of this PR and have a wider discussion.

Reply:
yes, makes sense, will revert this